### PR TITLE
Add fipy recipe

### PR DESCRIPTION
**Pull request by guyer**

### CODE REVIEW

**Comment by guyer** [comment]
@conda-forge/staged-recipes please consider this feedstock

File: recipes/fipy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    - weave       # [not py3k]
+
+test:
+  imports:
+    - fipy
+
+about:
+  home: http://www.ctcms.nist.gov/fipy
+  license: NIST public domain
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.rst'
```
Comment (chrisburr) [inline recipes/fipy/meta.yaml]: The `RECIPE_DIR` part isn't needed
```suggestion
  license_file: LICENSE.rst
```

File: recipes/fipy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    - python
+    - pip
+    - numpy
+    - scipy
+    - setuptools
+  run:
+    - python
+    - numpy
+    - scipy
+    - gmsh <4.0   # [not (win and (py2k or x86))]
```
Comment (chrisburr) [inline recipes/fipy/meta.yaml]: It's not necessarily wrong but this selector looks questionable. Can you check it?

Comment (guyer) [reply recipes/fipy/meta.yaml]: Sadly, it's right. gmsh 4.x does not work at all. gmsh 3.x works on linux and macos. gmsh 3.x only works on win for py3k and x64.

File: recipes/fipy/LICENSE.rst [hunk 3]
```diff
@@ -0,0 +1,32 @@
+------------
```
Comment (isuruf) [inline recipes/fipy/LICENSE.rst]: Is this file in the source tarball? If so, you don't need this in the recipe

Comment (guyer) [reply recipes/fipy/LICENSE.rst]: It is in the tarball. I've removed `LICENSE.rst`.

File: recipes/fipy/meta.yaml [hunk 4]
```diff
@@ -42,7 +42,6 @@ test:
 about:
   home: http://www.ctcms.nist.gov/fipy
   license: NIST public domain
-  license_file: LICENSE.rst
```
Comment (chrisburr) [inline recipes/fipy/meta.yaml]: You still need the `licence_file` entry, you just don't need a copy of the file itself if it's in the tarball as conda will automatically look in the source tarball for it.

Comment (guyer) [reply recipes/fipy/meta.yaml]: Yeah, I realized that right after I posted. Fixed now.

**Comment by guyer** [comment]
@conda-forge/staged-recipes: conda-forge makes me sad. 
Can somebody please merge my feedstock?

**Pull request merged**